src/util.h

```diff
@@ -202,6 +202,7 @@ class ArgsManager
     void ParseParameters(int argc, const char*const argv[]);
     void ReadConfigFile(const std::string& confPath);
     std::vector<std::string> GetArgs(const std::string& strArg);
+    std::vector<std::string> GetUniqueArgs(const std::string& strArg);
```

Comment:
There are other multiargs that may use this getter, but I'm checking it. There is also the option of `GetArgs(const std::string& strArg, fUnique = false)`.